StyleCop.Analyzers/StyleCop.Analyzers/Helpers/XmlSyntaxFactory.cs

```diff
@@ -171,6 +171,11 @@ public static XmlElementSyntax SummaryElement(string newLineText, SyntaxList<Xml
             return MultiLineElement(XmlCommentHelper.SummaryXmlTag, newLineText, content);
         }
 
+        public static XmlElementSyntax EmptySummaryElement(string newLineText)
```

Comment:
💭 The behavior of `SummaryElement(newLineText)` should be acceptable for an empty element. It will produce content with the following form:

```csharp
/// <summary>
/// 
/// </summary>
```

Conveniently this is the same form you get if you manually type `///` in the editor.